crates/database/src/orders.rs

```diff
@@ -781,14 +788,66 @@ mod tests {
         assert!(get_order(&mut db, 2).await.is_some());
     }
 
+    // This test is only for the PR. Can be removed before merging...
+    #[tokio::test]
+    #[ignore]
+    async fn postgres_user_orders_performance() {
+        let mut db = PgConnection::connect("postgresql://").await.unwrap();
+        let mut db = db.begin().await.unwrap();
+        crate::clear_DANGER_(&mut db).await.unwrap();
+
+        type Data = ([u8; 56], Address, DateTime<Utc>);
+        async fn user_orders(
+            ex: &mut PgConnection,
+            owner: &Address,
+            offset: i64,
+            limit: Option<i64>,
+        ) -> Vec<Data> {
+            super::user_orders(ex, owner, offset, limit)
+                .map(|o| {
+                    let o = o.unwrap();
+                    (o.uid.0, o.owner, o.creation_timestamp)
+                })
+                .collect::<Vec<_>>()
+                .await
+        }
+
+        // fill the database
+        for i in 0..240u32 {
+            let mut owner_bytes = i.to_ne_bytes().to_vec();
+            owner_bytes.append(&mut vec![0; 20 - owner_bytes.len()]);
+            let owner = ByteArray(owner_bytes.try_into().unwrap());
+            for j in 0..1000u32 {
+                let mut i_as_bytes = i.to_ne_bytes().to_vec();
+                let mut j_as_bytes = j.to_ne_bytes().to_vec();
+                let mut order_uid_info = vec![0; 56 - i_as_bytes.len() - j_as_bytes.len()];
+                order_uid_info.append(&mut j_as_bytes);
+                i_as_bytes.append(&mut order_uid_info);
+                let order = Order {
+                    owner,
+                    uid: ByteArray(i_as_bytes.try_into().unwrap()),
+                    creation_timestamp: Utc::now(),
+                    ..Default::default()
+                };
+                insert_order(&mut db, &order).await.unwrap();
+            }
+        }
+
+        let now = Utc::now();
+        let _result = user_orders(&mut db, &ByteArray([2u8; 20]), 10, Some(10)).await;
+        let time_diff = Utc::now() - now;
+        println!("{:?}", time_diff);
+        assert!(time_diff < chrono::Duration::seconds(1));
```

Comment:
```suggestion
        let now = std::time::Instant::now();
        let _result = user_orders(&mut db, &ByteArray([2u8; 20]), 10, Some(10)).await;
        let time_diff = now.elapsed();
        println!("{:?}", elapsed);
        assert!(elapsed < std::time::Duration::from_secs(1));
```
Prefer std types when they accomplish the same thing.